tools/public_api_guard/material/datepicker.md

```diff
@@ -107,6 +108,16 @@ export type ExtractDateTypeFromSelection<T> = T extends DateRange<infer D> ? D :
 // @public
 export const MAT_DATE_RANGE_SELECTION_STRATEGY: InjectionToken<MatDateRangeSelectionStrategy<any>>;
 
+// @public
+export const MAT_DATEPICKER_INTL_PROVIDER: {
```

Comment:
I don't think that we should expose this and the factory as a public API.

Reply:
Thanks @crisbeto  for your feedback . I have taken note that changes are only required in the docs area. I have submitted a new PR #26738  for review.